optimum/utils/input_generators.py

```diff
@@ -97,6 +97,10 @@ def __getattr__(self, attr_name):
         return super().__getattr__(attr_name)
 
 
+class NormalizedAudioConfig(NormalizedConfig):
```

Comment:
I guess it's not needed if it does not provide anything new.

Reply:
yes removed